CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Add a new console script that creates a template repository for an assignment (#91, @kcranston)
```

Comment:
just a note that we've been using the ISSUE number rather than the pr number given an issue could have a few pr's associated with it. and an issue in theory has more description of the actual item being addressed.